### PR TITLE
Added pickups

### PR DESCRIPTION
**Pull request by quickdevel**

### CODE REVIEW

**Comment by Nacorpio** [comment]
We should add an enum for different types of pickups.

File: source/Pickup.hpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "Entity.hpp"
+
+namespace GTA
+{
+	public ref class Pickup sealed : public Entity
```
Comment (zorg93) [inline source/Pickup.hpp]: Pickups are not entities, Dont inherit the entity class into pickup class

Comment (quickdevel) [inline source/Pickup.hpp]: I tested Entity-class with ambient pickups and it worked. I'am sorry. :P

Comment (crosire) [inline source/Pickup.hpp]: If all Entity related functions work on pickups too, then that can very well be. But only then =).

**Pull request merged**